code/__defines/feedback.dm

```diff
@@ -0,0 +1,5 @@
+/// Generic feedback failure message handler.
+#define FEEDBACK_FAILURE(USER, MSG) to_chat(USER, SPAN_WARNING(MSG))
+
+/// Feedback messages intended for use in `use_*` overrides. These assume the presence of the `user` variable.
+#define USE_FEEDBACK_FAILURE(MSG) FEEDBACK_FAILURE(user, MSG)
```

Comment:
I do, yes. It's part of a larger project I have in mind to standardize message usage since they're currently inconsistent, but I figured I would introduce this now for the common macro used here.

Reply:
Fine in my book then